client/src/search.tsx

```diff
@@ -337,7 +337,8 @@ function InnerSearchNavigateWidget(props: InnerSearchNavigateWidgetProps) {
                 {...getItemProps({
                   key: item.url,
                   className:
-                    "result-item " + (i === highlightedIndex ? "highlit" : ""),
+                    "result-item " +
+                    (i === highlightedIndex ? "highlight" : ""),
```

Comment:
I renamed the CSS as well, but I wasn't sure if this was intentional for a different reason